### PR TITLE
Add Reading List API reference

### PR DESCRIPTION
**Pull request by oliverdunk**
Adds documentation for the new [`chrome.readingList`](https://github.com/w3c/webextensions/blob/main/proposals/reading_list.md) API.

Sample: https://github.com/GoogleChrome/chrome-extensions-samples/pull/1021
Related: https://github.com/GoogleChromeLabs/Extension-Docs/issues/174

### CODE REVIEW

File: site/en/docs/extensions/reference/readingList/index.md
Comment (AmySteam) [inline site/en/docs/extensions/reference/readingList/index.md:6]:
```suggestion
Chrome features a reading list located on the side panel. It lets users save web pages to read later or when offline.
```

File: site/en/docs/extensions/reference/readingList/index.md
Comment (AmySteam) [inline site/en/docs/extensions/reference/readingList/index.md:7]:
```suggestion
Use the Reading List API to retrieve existing items and add or remove items from the list.
```

File: site/en/docs/extensions/reference/readingList/index.md
Comment (AmySteam) [inline site/en/docs/extensions/reference/readingList/index.md:32]: Here's an idea, since the concepts are short and few, what do you think about moving them into the use cases as an aside or within the content. 

For example:
"Items in the reading list are not in any guarenteed order." could go as a note under **display items**
 
And "Items are keyed by URL. This includes the hash and query string." Could go under **Add an item**.

Thoughts?

File: site/en/docs/extensions/reference/readingList/index.md
Comment (AmySteam) [inline site/en/docs/extensions/reference/readingList/index.md:60]:
```suggestion
To display items from the reading list, use the [`chrome.readingList.query()`][query] to retrieve them.
```

File: site/en/docs/extensions/reference/readingList/index.md
Comment (AmySteam) [inline site/en/docs/extensions/reference/readingList/index.md:73]:
```suggestion
You can use [`chrome.readingList.updateEntry`][update-entry] to update the title, URL, and reading status. The following code marks an item as read:
```

**Review by AmySteam — CHANGES_REQUESTED** [review]
@oliverdunk Thanks for your work on this. I think it's a great MVP. I just added a couple of suggestions and I wanted to explore an idea for the conceptual information.

I will approve this once the sample PR is merged. 

Thanks again 💯

**Comment by AmySteam** [comment]
Last request:
Can you also add this permission to the general permissions page https://developer.chrome.com/docs/extensions/mv3/declare_permissions/?

**Comment by AmySteam** [comment]
@oliverdunk Can you make this a priority tomorrow? I'd like to merge this tomorrow in time for the Chrome 120 beta release.

File: site/en/docs/extensions/reference/readingList/index.md
Comment (oliverdunk) [reply site/en/docs/extensions/reference/readingList/index.md:32]: I think there is value to keeping them here actually. I feel like they are useful things you should know before doing anything with the API, vs things you only learn if you look at the "Add an item" section for example.

Comment (AmySteam) [reply site/en/docs/extensions/reference/readingList/index.md:32]: I understand your logic, so let's go with that. Visually, I think it looks pretty empty, maybe a definition list would help. I'm going to approve since it's not a deal-breaker

**Review by AmySteam — APPROVED** [review]
LGTM 👍 Thanks Oliver! 💯

**Pull request merged**
